### ISSUE TITLE
tightening up definition of arrest

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#2850]
I suspect we (maybe just me) have been a bit sloppy in our use of arrest.

I realised this from a conversation with Jacky this week, which resulted in my needing to check these:
https://github.com/pombase/curation/issues/1100

Then, a paper I am reading today for another ticket highlighted the problem for me:

"Single cs top2 cells are not appropriate for identifying the blocked stage in chromosome structural changes be- cause other mitotic events proceed independently (see Introduction). "

and in the introduction
"age topo II becomes essential in S. pombe. Single ts fop2 mutants of S. pombe are not appropriate for examin- ing the blocked stages in chromosome structural changes because the mutant phenotypes are obscured by other mitotic events, including spindle dynamics, septum forma- tion, and cytokinesis, that take place in top2 cells. The structurally aberrant chromosomes are transiently pulled by the spindle but fail to separate (uncoordinated mitosis) (Uemura and Yanagida, 1986; Hirano et al., 1986), and the undivided nuclei are cut across by the septum and broken by cytokinesis (Uemura and Yanagida, 1984, 1988). For this reason, we isolated and characterized a cs top2 mu- tant in order to construct a double cs top2-cs nda3 mu- tant, which is expected to be blocked both in spindle for- mation and cytokinesis. Indeed, the double cs mutant was arrested at an earlier stage than that blocked by a single cs nda3 mutant. "

So, I think we can tighten the defs so that we only say it is arrest IF the cells arrest  uniformly with characteristics of a specific cell cycle stage (  should also always specify whether reversible?)

**Comment by ValWood** [comment]
(i.e. it isn't arrest if other events proceed independently)

**Comment by ValWood** [comment]
We should add the characteristics to the arrest terms too to make it easier. Even experts get confused

like prophase "uncondensed chromsomes"
metaphase "condensed unseparated chromosomes, short spindle"
etc

**Comment by mah11** [comment]
> (i.e. it isn't arrest if other events proceed independently)

OK, I can make changes consistent with this, but in this 

> we only say it is arrest IF the cells arrest uniformly with characteristics of a specific cell cycle stage

I don't understand what you mean by "cells arrest uniformly". Could you clarify?


> We should add the characteristics to the arrest terms too to make it easier.

Good idea, but I will need a comprehensive list of features to use.

**Comment by ValWood** [comment]
Its OK this def says it
A cell cycle phenotype that affects or involves the normal or abnormal occurrence of cell cycle arrest. In cell cycle arrest, progression through the cell cycle is halted during one of the normal phases (G1, S, G2, or M).

I just wanted to be sure that people only use arrest when progression is halted in a specific phase. I am sure I have seen "arrest" used incorrectly when cell death is not at a specific phase 
(maybe not).

To make sure it is used correctly we can block all of the terms  which do not specify a phase perhaps?

I notice that FYPO:0001024
already has the "not for annotation" comment. But we should be able to add to
FYPO:0000407
FYPO:0001026
FYPO:0000502
possibly others?

**Comment by ValWood** [comment]
> We should add the characteristics to the arrest terms too to make it easier.
Good idea, but I will need a comprehensive list of features to use.

We can think about these (or ask @jvhayles)

**Comment by mah11** [comment]
> I notice that FYPO:0001024 already has the "not for annotation" comment. But we should be able to add to FYPO:0000407, FYPO:0001026, FYPO:0000502, [and] possibly others?

Would a soft check do for these?

**Comment by ValWood** [comment]
Soft check is a good start. Can always be blocked later...

**Comment by mah11** [comment]
I've added those 3 terms to FYPO_terms_excluded_from_pombase.txt. Add whichever others you find ...

Is there anything else for this ticket?

**Comment by mah11** [comment]
closing because I think we've got it more or less working

if you get info to put in comments for more terms (prophase and metaphase are done ), open a new ticket
